build.gradle

```diff
@@ -53,6 +53,8 @@ dependencies {
     compile 'org.springframework.boot:spring-boot-starter-data-jpa'
     // https://mvnrepository.com/artifact/org.apache.commons/commons-lang3
     compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.4'
+    // https://mvnrepository.com/artifact/org.apache.commons/commons-email
+    compile group: 'org.apache.commons', name: 'commons-email', version: '1.2'
```

Comment:
i've been finding apache's libraries have a cleaner api and are easier to use than some of the other ones. for example i switched email libs because the other one didn't seem to have a way to send an html message (probably just couldn't find out how)